Source/Core/DolphinQt2/HotkeyScheduler.cpp

```diff
@@ -159,17 +159,17 @@ void HotkeyScheduler::Run()
       if (IsHotkey(HK_EXIT))
         emit ExitHotkey();
 
+      auto& settings = Settings::Instance();
+
       // Volume
       if (IsHotkey(HK_VOLUME_DOWN))
-        AudioCommon::DecreaseVolume(3);
+        settings.SetVolume(settings.GetVolume() - 3);
 
       if (IsHotkey(HK_VOLUME_UP))
-        AudioCommon::IncreaseVolume(3);
+        settings.SetVolume(settings.GetVolume() + 3);
 
       if (IsHotkey(HK_VOLUME_TOGGLE_MUTE))
-        AudioCommon::ToggleMuteVolume();
-
-      auto& settings = Settings::Instance();
+        settings.SetVolume(0);
```

Comment:
This is not the same as muting the audio (though it probably should be? Not sure why AudioCommon treats the two differently).